------------------------------------------------------------------------
r234743 | rmh | 2012-04-28 00:27:21 +0200 (ds, 28 abr 2012) | 5 lines

Increase DFLDSIZ from 128 MiB to 32 GiB.  On amd64 there's plenty of virtual
memory available, so there is no need to be so conservative about it.

Reviewed by:    arch

------------------------------------------------------------------------

--- a/sys/amd64/include/vmparam.h
+++ b/sys/amd64/include/vmparam.h
@@ -54,7 +54,7 @@
  */
 #define	MAXTSIZ		(128UL*1024*1024)	/* max text size */
 #ifndef DFLDSIZ
-#define	DFLDSIZ		(128UL*1024*1024)	/* initial data size limit */
+#define	DFLDSIZ		(32768UL*1024*1024)	/* initial data size limit */
 #endif
 #ifndef MAXDSIZ
 #define	MAXDSIZ		(32768UL*1024*1024)	/* max data size */
